docs/src/main/sphinx/installation/plugins.md

```diff
@@ -0,0 +1,284 @@
+# Plugins 
+
+Trino uses a plugin architecture to extend its capabilities and integrate with
+various data sources and other systems. Trino includes many plugins as part of
+the binary packages - specifically the `tar.gz` archive and the Docker
+container.
+
+The main plugins implement some the following functionalities:
+
+* [Connectors](/connector)
+* [Access control systems](security-access-control)
+* [Event listeners](admin-event-listeners)
+* Additional [types](/develop/types) and [functions](/develop/functions)
+
+All plugins are optional for your use of Trino since they support a specific
+functionality, that is potentially not needed for your use. Plugins are located
+in the `plugin` folder of your Trino installation, and are loaded automatically
+during Trino startup.
+
+(plugins-downloading)=
+## Downloading
+
+Typically downloading a plugin is not necessary, since Trino binaries include
```

Comment:
"Typically, downloading a plugin is not necessary because..."

Reply:
updated